### DISCUSSION TITLE
I tried this in Swift

### DISCUSSION THREAD

**Discussion by iankoex** [supabase/supabase#1415] [Feature Requests]
what am I doing wrong? 
The code doesn't sign up the user. 
![Screenshot](https://user-images.githubusercontent.com/30172987/116994225-1c754a80-ace1-11eb-882a-da5c1ecfee44.png)

**Comment by silentworks** [comment]
Is this related to Supabase? If yes can you provide the endpoint url that your `request` is posting to please? Also, can you state what error you are getting if any or what do you see when you make the request?

**Reply by iankoex** [reply]
Yes. It is related to Supabase. I was trying to use Supabase in swift to create an iOS swiftUI app. There is no way I can easily do this because of lack of a swift client library so I decided to create one. I went trough the gotrue C# code and saw that it was essentially using http to sign up users by passing the Supabase key as a header and the username and password as data. Trying the same in swift it doesnt work.
The url is the Supabase url then add /signup ; it is used this way in the gotrue C# code.
I will share the errors when i get to my computer. As soon as I can.

**Reply by iankoex** [reply]
![Screenshot 2021-05-05 at 22 10 50](https://user-images.githubusercontent.com/30172987/117196888-c4793980-adef-11eb-9083-dece48c1e5cf.png)
here is the 'error' i pressed the button a couple of times.

**Reply by silentworks** [reply]
I'm not a swift developer so not sure what those errors mean. What is the `url` you are making the request to?
